### PR TITLE
Fixes #134 - Ask on uninstall procedure

### LINKED ISSUE #134: Ask on uninstall to delete all data from DB
Same as we have in TYL plugin. When you click on uninstall button you are asked if you wanna drop all DB data etc. If you select no, this is the easy way how to reinstall without data loss. Also this could be a prevention to lost data.

### PR DESCRIPTION
**Pull request by Eldenroot**
Resolves #134

### CODE REVIEW

**Comment by Eldenroot** [comment]
Maybe the text could be improved (I am not a native English speaker)

File: upload/inc/languages/english/admin/isango.lang.php [hunk 1]
```diff
@@ -29,3 +29,6 @@
 $l['isango_input_mode_max'] = 'Big size buttons with text';
 $l['isango_input_mode_min'] = 'Small buttons with icon only';
 $l['isango_input_mode_pop'] = 'A select dropdown';
+
+$l['isango_uninstall'] = 'Isango - uninstallation';
+$l['isango_uninstall_message'] = 'Do you wish to drop ALL plugin data from the database? Selecting \"No\" will leave untouched:<ul><li>Linked accounts for users.</li>\n\n</ul>\nSelecting \"No\" will <em>not</em>, however, prevent the removal of:<ul><li>Plugin settings (including any saved settings).</li>\n<li>The plugin\'s stylesheet, \"isango.css\" (including any changes you\'ve made to it), accessible for each theme via the ACP\'s Templates & Style -> <a href=\"index.php?module=style-themes\">Themes</a> module.</li>\n<li>The plugin\'s templates (including any changes you\'ve made to them), accessible  under each template set via the ACP\'s Templates & Style -> <a href=\"index.php?module=style-templates\">Templates</a> module.</li>\n</ul>';
```
Comment (effone) [inline upload/inc/languages/english/admin/isango.lang.php]: There are quotes and escape issues in this language string. It appears like this:
image

File: upload/inc/languages/english/admin/isango.lang.php [hunk 2]
```diff
@@ -29,3 +29,6 @@
 $l['isango_input_mode_max'] = 'Big size buttons with text';
 $l['isango_input_mode_min'] = 'Small buttons with icon only';
 $l['isango_input_mode_pop'] = 'A select dropdown';
+
+$l['isango_uninstall'] = 'Isango - uninstallation';
```
Comment (effone) [inline upload/inc/languages/english/admin/isango.lang.php]: Set capital `U` for uninstallation. Remove hyphen (-) to match with MyBB Title Style

Comment (Eldenroot) [reply upload/inc/languages/english/admin/isango.lang.php]: Done!

**Review by effone — APPROVED** [review]

**Pull request merged**